src/components/results.js

```diff
@@ -89,10 +89,27 @@ const ResultsComponent = React.createClass({
           block.userId
         }/${d._id}/raw/${block.thumb}/thumbnail.png)`
       } else if (screenshots.indexOf(`${d._id}.png`) > -1) {
-        style.backgroundImage = `url(https://christopheviau.com/block_screenshot/${
-          d._id
-        }.png)`
-        classNameString = `${classNameString} ` + `no-thumbnail`
+        const chrisVStem = 'https://christopheviau.com'
+        const chrisVUrl = `${chrisVStem}/block_screenshot/${d._id}.png`
+
+        style.backgroundImage = `url(${chrisVUrl})`
+        classNameString = `${classNameString} no-thumbnail`
+      } else {
+        // construct a url for a thumbnail image
```

Comment:
Looks great! Nice and straightforward use of Google Cloud.

Reply:
yea, I thought so.  I like that I can make public bucket object urls over in my [screenshot-service](https://github.com/micahstubbs/screenshot-service/blob/master/dedicated-server/cache.js#L26) server-side code, so that I don't need any special cloud-vendor nodejs library at all here in my client-side code.